apps/generic/main.cpp

```diff
@@ -42,40 +42,39 @@
 #include <QtDeclarative>
 #include <QScopedPointer>
 
-Q_DECL_EXPORT int main(int argc, char *argv[])
+int main(int argc, char** argv)
 {
     QString language;
+    QApplication application(argc, argv);
 
-// Load language file depending on platform
+	// Load language file depending on platform
 #if defined Q_WS_MAEMO_5
-    QApplication *app = new QApplication( argc, argv );
     language = QString(getenv("LC_NAME")).split('_').first();
     qDebug() << "language:" << language << getenv("LANG") << getenv("LC_NAME");
 #else
-    QApplication *app = new QApplication( argc, argv );
     language = QLocale::system().name();
-//    language = QString(getenv("LC_NAME")).split('_').first();
+    qDebug() << "got language:" << language;
 #endif
 
-
     QTranslator qtTranslator;
+
     if(!qtTranslator.load("qt_" + language, QLibraryInfo::location(QLibraryInfo::TranslationsPath))) {
         qDebug() << "couldn't load qt_" + language;
     }
-    app->installTranslator(&qtTranslator);
+    application.installTranslator(&qtTranslator);
 
     QTranslator translator;
     if (!translator.load(":/kodimote_" + language + ".qm")) {
         qDebug() << "Cannot load translation file" << "kodimote_" + language + ".pm";
     }
-    app->installTranslator(&translator);
+    application.installTranslator(&translator);
 
     // Load debug stuff
     XDebug::addAllowedArea(XDAREA_GENERAL);
```

Comment:
Indeed... this should be koDebug(), however, it's not this branch's fault. I seem to have overlooked something when doing the rename. I'll clean it up myself. No need to open that can of worms in this branch.